Added more use-cases to useHealIfRelevant

sktan opened the pull request:
Added Pumped Up to UseHealIfRelevant because that also heals us back to full health.
Also uses STEAL_HEALTH as a last resort if we have no other healing methods remaining.

sktan committed:
Added more use-cases to useHealIfRelevant

Added Pumped Up to UseHealIfRelevant because that also heals us back to full health.
Also uses STEAL_HEALTH as a last resort if we have no other healing methods remaining.

ghost commented:
I think it would be best to use pumped up before medic, as it is basically a better medic. The health increase I found to be almost negligible.

sktan committed:
Changed the ordering of healing priorities

Changed the order of healing priorities

1. Pumped Up
2. Healing Potion
3. God Mode
4. Steal Health

sktan commented:
Changed the ordering of priorities
